---
title: "Physcraper: A Python package for continual update of evolutionary estimates using the Open Tree of Life"

summary: "In this paper we present Physcraper, a Python package for update of phylogenetic relationships using existing expert-curated alignments and molecular data from GenBank that has not been analyzed in an evolutionary context yet." 

abstract: "1. Phylogenies are a key part of research in many areas of biology. Tools that automate some parts of the process of phylogenetic reconstruction, mainly molecular character matrix assembly, have been developed for the advantage of both specialists in the field of phylogenetics and nonspecialists. However, interpretation of results, comparison with previously available phylogenetic hypotheses, and selection of one phylogeny for downstream analyses and discussion still impose difficulties to one that is not a specialist either on phylogenetic methods or on a particular group of study.
2. Physcraper is a command-line Python program that automates the update of published phylogenies by adding public DNA sequences to underlying alignments of previously published phylogenies. It also provides a framework for straightforward comparison of published phylogenies with their updated versions, by leveraging upon tools from the Open Tree of Life project to link taxonomic information across databases.
3. Physcraper can be used by the nonspecialist, as a tool to generate phylogenetic hypotheses based on publicly available expert phylogenetic knowledge. Phylogeneticists and taxonomic group specialists will find it useful as a tool to facilitate molecular dataset gathering and comparison of alternative phylogenetic hypotheses (topologies).
4. The Physcraper workflow demonstrates the benefits of doing open science for phylogenetics, encouraging researchers to strive for better sharing practices. Physcraper can be used with any OS and is released under an open-source license. Detailed instructions for installation and use are available at https://physcraper.readthedocs."

authors:
- admin
- Martha Kandziora
- Emily Jane McTavish

date: "2020-09-01T00:00:00Z"

doi: "10.1101/2020.09.15.299156"

featured: false

image:
  caption: ''
  focal_point: ""
  preview_only: false
  
projects: []

publication: '*BioRxiv*(1)'

publication_short: ""

publication_types:
- "3"

publishDate: "2020-09-01T00:00:00Z"
# slides: example # This slideshow is in the dir /content/slides/example If you want another slide showm just change the name of the folder where the slideshow is in here!  

tags:
- Phylogenetic estimates
- Open science
- Python package

# words in title cannot be formatted as italics by using markdown; you can do it by changing some specs, see https://github.com/gcushen/hugo-academic/issues/280 for problems related to changing those specs.

url_code: ""
url_dataset: ""
url_pdf: https://www.biorxiv.org/content/10.1101/2020.09.15.299156v1.abstract
url_poster: ""
url_project: ""
url_slides: ""
url_source: ""
url_video: ""
---

<!--{{% alert note %}}
Click the *Cite* button above to demo the feature to enable visitors to import publication metadata into their reference management software.
{{% /alert %}}

{{% alert note %}}
Click the *Slides* button above to demo Academic's Markdown slides feature.
{{% /alert %}}

Supplementary notes can be added here, including [code and math](https://sourcethemes.com/academic/docs/writing-markdown-latex/).
-->
